core/src/main/java/io/camunda/migrator/RuntimeMigrator.java

```diff
@@ -74,14 +79,18 @@ public void migrate() {
       if (deployResourceCommandStep2 != null) {
         deployResourceCommandStep2.send().join();
       }
+      LOGGER.debug("Completed auto deployment.");
     }
 
     String latestLegacyId = idKeyMapper.findLatestIdByType("runtimeProcessInstance");
+    LOGGER.debug("Latest legacy id is {}", latestLegacyId);
     ProcessInstanceQuery processInstanceQuery = ((ProcessInstanceQueryImpl) runtimeService.createProcessInstanceQuery())
         .idAfter(latestLegacyId)
         .rootProcessInstances();
 
     long maxLegacyProcessInstanceCount = processInstanceQuery.count();
+    LOGGER.debug("Number of legacy process instances to migrate is {}", maxLegacyProcessInstanceCount);
```

Comment:
🙃 Just an idea.
```suggestion
    LOGGER.debug("{} process instances will be migrated", maxLegacyProcessInstanceCount);
```
or:
```suggestion
    LOGGER.debug("{} process instances are scheduled for migration", maxLegacyProcessInstanceCount);
```